src/main/java/com/adobe/epubcheck/opf/OPFChecker30.java

```diff
@@ -492,7 +492,7 @@ public static boolean isCommonVideoType(String type)
   public static boolean isBlessedFontType(String type)
   {
     return type.equals("application/vnd.ms-opentype") || type.equals("application/font-woff")
-        || type.equals("image/svg+xml");
+        || type.equals("image/svg+xml") || type.startsWith("font/");
```

Comment:
Don't we also need to add `application/font-sfnt` to the list?

Reply:
Ah, good point. I've changed the woff test to startsWith "application/font-" just to be liberal (PFR fonts could always rise from the dead).